starter-core/src/main/java/io/micronaut/starter/feature/dependencies/DependenciesConfiguration.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2020 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.starter.feature.dependencies;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import edu.umd.cs.findbugs.annotations.Nullable;
+import io.micronaut.context.annotation.EachProperty;
+import io.micronaut.context.annotation.Parameter;
+
+@EachProperty("dependencies")
+public class DependenciesConfiguration implements MavenCoordinate {
```

Comment:
Instead of having this class that implements the interface you could annotate the interface directly. That works for `@ConfigurationProperties` so I think it should work with `@EachProperty`

Reply:
This no longer applies